### PR TITLE
Unpin sphinx from 2.4.4

### PR DESCRIPTION
**Pull request by dannysepler**
According to #3506 , 3.0.0 broke the docs, but 3.0.1 fixed the docs again. Pinning to 3.1, just to be safe. I clicked around locally and things seem to work.

### CODE REVIEW

**Comment by scopatz** [comment]
Thanks @dannysepler! No news item needed here

File: requirements/docs.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cloud_sptheme
 numpydoc
-Sphinx==2.4.4
+Sphinx>=3.1
```
Comment (scopatz) [inline requirements/docs.txt:3]: should the be `>=3.1,<3`?

Comment (AstraLuma) [reply requirements/docs.txt:3]: pretty sure it's an AND?

Comment (dannysepler) [reply requirements/docs.txt:3]: right! it would fail on that i think?

Comment (scopatz) [reply requirements/docs.txt:3]: Ahh OK, this is fine then

**Pull request merged**